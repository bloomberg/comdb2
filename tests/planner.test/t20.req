explain query plan SELECT cusip, trade_date, ticker, coupon, coupon_type, bond_type, maturity, issuer_id, mdy_rating, sp_rating,           rating_category, industry, eqty_ticker, eqty_price, cv_ratio, trac_size, trac_price, trac_yield, trac_spread, trac_z_spread, trac_i_spread,       trac_asw, trac_oas, trac_trade_time,trac_tot_volume, trac_total_trades, flags, trac_oas_vol, bench_cusip, trac_is_plus, bi_industry, rps_all,     trac_g_spread, ntrd_1mm_buy,ntrd_1mm_sell,ntrd_1mm_dlr,ntrd_5mm_buy,ntrd_5mm_sell,ntrd_5mm_dlr , is_144a  FROM AGGRQUOTES WHERE  ( bond_type=1    or  bond_type=5)  and ( (sp_rating >= 1 and sp_rating <= 1)  or  (mdy_rating >= 1 and mdy_rating <= 1) ) and (maturity >= 20141203 and maturity   <= 21101203)  and trac_total_trades > 0 and (  trade_date=20141117 or  trade_date=20141118 );
