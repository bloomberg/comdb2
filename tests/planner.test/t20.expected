(selectid=0, order=0, from=0, detail='EXECUTE LIST SUBQUERY 1')
(selectid=0, order=0, from=0, detail='EXECUTE LIST SUBQUERY 1')
(selectid=0, order=0, from=0, detail='SEARCH TABLE AGGRQUOTES USING INDEX aggrquotes_ix_8 (trade_date=? AND bond_type=? AND sp_rating>? AND sp_rating<?) (88 rows)')
(selectid=0, order=0, from=0, detail='EXECUTE LIST SUBQUERY 1')
(selectid=0, order=0, from=0, detail='EXECUTE LIST SUBQUERY 1')
(selectid=0, order=0, from=0, detail='SEARCH TABLE AGGRQUOTES USING INDEX aggrquotes_ix_7 (trade_date=? AND bond_type=? AND mdy_rating>? AND mdy_rating<?) (88 rows)')
